ClearKeep/Scenes/Home/Modifiers/InCallModifier.swift

```diff
@@ -0,0 +1,284 @@
+					}
+					
+					isInMinimizeMode = false
+					
+					let viewController = UIHostingController(rootView: InCallView(viewModel: callViewModel))
+					viewController.modalPresentationStyle = .overFullScreen
+					callController = viewController
+					let sceneDelegate = UIApplication.shared.connectedScenes.first?.delegate as? SceneDelegate
+					sceneDelegate?.window?.rootViewController?.present(viewController, animated: true)
+				}
+				.transition(.move(edge: .top))
+				
+				if isInMinimizeMode {
+					mimimumView
+				}
+			}
+		}
+		.onReceive(NotificationCenter.default.publisher(for: NSNotification.Name.CallService.receiveCall)) { _ in
+			callViewModel.backHandler = {
+				callController?.dismiss(animated: true)
+				
+				withAnimation {
+					isInCall = true
+					isInMinimizeMode = true
+				}
+			}
+			
+			self.callViewModel.timeCounter.sec = 0
+			self.callViewModel.timeCounter.min = 0
+			self.callViewModel.timeCounter.hour = 0
+			self.callViewModel.callInterval = 0
+			let viewController = UIHostingController(rootView: InCallView(viewModel: callViewModel))
+			callController = viewController
+			viewController.modalPresentationStyle = .overFullScreen
+			let sceneDelegate = UIApplication.shared.connectedScenes.first?.delegate as? SceneDelegate
+			sceneDelegate?.window?.rootViewController?.present(viewController, animated: true)
+		}
+		.onReceive(NotificationCenter.default.publisher(for: NSNotification.Name.CallService.endCall)) { _ in
+			isInCall = false
+			isInMinimizeMode = false
+			self.callViewModel.speakerEnable = false
+			self.callViewModel.microEnable = true
+			self.callViewModel.cameraFront = false
+			self.callViewModel.localVideoView = nil
+			self.callViewModel.remoteVideoView = nil
+			self.callViewModel.remotesVideoView.removeAll()
+			if var topController = UIApplication.shared.keyWindow?.rootViewController {
+				while let presentedViewController = topController.presentedViewController {
+					topController = presentedViewController
+				}
+
+				topController.dismiss(animated: true)
+			}
+		}
+	}
+	
+	var mimimumView: some View {
+		VStack {
+			HStack(alignment: .top) {
+				Spacer()
+				if callViewModel.callBox?.isCallGroup ?? false {
+					ZStack {
+						LinearGradient(gradient: Gradient(colors: AppTheme.shared.colorSet.gradientPrimary), startPoint: .leading, endPoint: .trailing)
+							.frame(maxWidth: .infinity, maxHeight: .infinity)
+
+						VStack {
+							Text(callViewModel.getUserName())
+								.font(AppTheme.shared.fontSet.font(style: .body2))
+								.foregroundColor(.white)
+								.frame(maxWidth: .infinity)
+								.padding(10)
+						}
+					}
+					
+				} else if callViewModel.callType == .video && callViewModel.cameraOn, let videoView = callViewModel.localVideoView {
+					VideoView(rtcVideoView: videoView)
+				} else {
+					ZStack {
+						if let avatar = callViewModel.callBox?.avatar,
+						   !avatar.isEmpty {
+							AsyncImage(url: URL(string: avatar)) { image in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/Home/Modifiers/InCallModifier.swift#L170"></span>Arguments should be either on the same line, or one per line.<br />
<code>multiline_arguments</code> <code>InCallModifier.swift:170</code></td>
    </tr>
  </tbody>
</table>